fetch.bs

```diff
@@ -4151,7 +4174,8 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    `<code>If-Range</code>`, then set <var>httpRequest</var>'s
    <a for=request>cache mode</a> to "<code>no-store</code>".
 
-   <li><p>If <var>httpRequest</var>'s <a for=request>cache mode</a> is "<code>no-cache</code>" and
+   <li><p>If <var>httpRequest</var>'s <a for=request>cache mode</a> is "<code>no-cache</code>",
+   <a for=request>prevent no-cache cache-control header modification flag</a> is unset and
```

Comment:
You need to lead this with _httpRequest_ again assuming that's where you grab the flag from.

Reply:
Also need an Oxford comma.

Reply:
done